sdktests/testapi_sdk_data.go

```diff
@@ -13,44 +13,84 @@ import (
 // SDKDataSource is a test fixture that provides a callback endpoint for SDK clients to connect to,
 // simulating the LaunchDarkly streaming or polling service.
 type SDKDataSource struct {
```

Comment:
I could have made separate SDKStreamingDataSource and SDKPollingDataSource types here, but the exact nature of the data source isn't super relevant in most of the tests— only when we're specifically testing streaming or polling behavior— so I felt like it's still nice to represent it more abstractly in this way.